src/SFA.DAS.ApprenticeFeedback.Domain/Models/ApprenticeFeedbackTarget.cs

```diff
@@ -33,7 +37,11 @@ public enum FeedbackTargetStatus
                 ApprenticeshipId = source.ApprenticeshipId,
                 Status = (FeedbackTargetStatus)source.Status,
                 StartDate = source.StartDate,
-                EndDate =source.EndDate
+                EndDate =source.EndDate,
```

Comment:
Did any unit tests need updating for this cast?

Reply:
Need help with 'WhenMappingFromApprenticeFeedbackEntityToModel'